src/lib/tls/asio/asio_stream.h

```diff
@@ -516,6 +537,18 @@ class Stream
 
       //! @}
 
+      // TODO: do we need these public? asio ssl stream doesn't have them, but they are very useful for testing
```

Comment:
as it says in the TODO

Reply:
"Testing" as in "Debugging"?
But the stream should just work (tm), right? :) I feel like if a user has to read those flags, they are probably deep enough in the code to just print them themselves, but I don't have a strong feeling 🤷‍♂

Reply:
No, I mean I find them really useful in `test_server_stream` in order to verify the `close_notify` was sent/received.